dirs/dirs.go

```diff
@@ -241,6 +242,7 @@ func SetRootDir(rootdir string) {
 	SnapNamesFile = filepath.Join(SnapCacheDir, "names")
 	SnapSectionsFile = filepath.Join(SnapCacheDir, "sections")
 	SnapCommandsDB = filepath.Join(SnapCacheDir, "commands.db")
+	SnapAuxStoreInfoDir = filepath.Join(SnapCacheDir, "store", "aux")
```

Comment:
Silly question (and nitpick!): why do we nest this? Could we use just `store` ? SnapStoreInfoDir and `/var/cache/snapd/store` ? Or do we expect that there will be more dirs beside `aux` ? If we want to nest it, would "extra" be appropriate for "aux" as well?

Reply:
actually we'll remove the /store/ level here, other things in that dir are from the store already